DigitalLearningSolutions.Web/Views/Shared/Components/NumericInput/Default.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents.DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents

```

Comment:
This is a knock-on effect from the weird namespaces. Overly long using statement that should be set back to normal when the view model is moved back to normal namespacing.